.github/workflows/ci.yml

```
@@ -20,10 +20,10 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - 3.6
           - 3.7
           - 3.8
           - 3.9
+          - 3.10
```

```suggestion
          - '3.10'
```

Let's try with a quote, because apparently YAML thinks it's a float... And people wonder why I hate YAML...